source/quickstart.rst

```diff
@@ -12,7 +12,11 @@
 
     This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath .NET SDK.
 
-.. only:: not (csharp or vbnet)
+.. only:: python
+
+    This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath Python SDK.
+
+.. only:: not (csharp or vbnet or python)
```

Comment:
I suppose so -- should we try to extract that out or wait until everyone gets theirs in? oO